miso-web/src/main/webapp/scripts/bulk_sample.js

```diff
@@ -1141,11 +1141,14 @@ BulkTarget.sample = (function ($) {
       );
 
       if (
-        (!Constants.isDetailedSample || show["Stock"] || show["Aliquot"]) &&
+        (!Constants.isDetailedSample ||
+          show["Stock"] ||
+          show["Aliquot"] ||
+          show["Tissue Processing"]) &&
         !config.isLibraryReceipt
       ) {
         columns = columns.concat(BulkUtils.columns.volume(true, config));
-        if (Constants.isDetailedSample) {
+        if (Constants.isDetailedSample && !show["Tissue Processing"]) {
```

Comment:
The `show[...]` values are true if the specified category is the target category (main sample being entered) OR if it's an upstream category that needs to be created as part of the hierarchy. e.g. `show["Tissue"]` is true when receiving stocks because we need to enter tissue information at the same time.

Here, you should check `targetCategory` instead because you only want to avoid showing `parentUsed` if the target category is "Tissue Processing."